google/cloud/documentai_toolbox/wrappers/page.py

```diff
@@ -44,37 +44,54 @@ class Table:
     _page: "Page" = dataclasses.field(repr=False)
 
     @cached_property
-    def body_rows(self):
-        return _table_rows_from_documentai_table_rows(
-            table_rows=list(self.documentai_object.body_rows),
-            text=self._page._document_text,
-        )
+    def body_rows(self) -> List[List[str]]:
+        return self._extract_table_rows(self.documentai_object.body_rows)
 
     @cached_property
-    def header_rows(self):
-        return _table_rows_from_documentai_table_rows(
-            table_rows=list(self.documentai_object.header_rows),
-            text=self._page._document_text,
-        )
+    def header_rows(self) -> List[List[str]]:
+        return self._extract_table_rows(self.documentai_object.header_rows)
 
     def to_dataframe(self) -> pd.DataFrame:
-        r"""Returns pd.DataFrame from documentai.table
+        """Returns pd.DataFrame from documentai.table
 
         Returns:
             pd.DataFrame:
                 The DataFrame of the table.
-
         """
         if not self.body_rows:
             return pd.DataFrame(columns=self.header_rows)
 
-        if self.header_rows:
-            columns = pd.MultiIndex.from_arrays(self.header_rows)
-        else:
-            columns = [None] * len(self.body_rows[0])
+        columns = (
+            pd.MultiIndex.from_arrays(self.header_rows)
+            if self.header_rows
+            else [None] * len(self.body_rows[0])
+        )
 
         return pd.DataFrame(self.body_rows, columns=columns)
 
+    def _extract_table_rows(
+        self, table_rows: Iterable[documentai.Document.Page.Table.TableRow]
+    ) -> List[List[str]]:
+        """Returns a list of rows from table_rows.
+
+        Args:
+            table_rows (List[documentai.Document.Page.Table.TableRow]):
+                Required. A documentai.Document.Page.Table.TableRow.
+
+        Returns:
+            List[List[str]]:
+                A list of table rows.
+        """
+        return [
+            [
+                _text_from_layout(cell.layout, self._page._document_text).replace(
+                    "\n", ""
```

Comment:
I know this already existed before this refactoring, but is removing all new lines (even if in the middle of the cell's content) what the user would want?  I imagine it is possible to have multi-line text in the headers.

Reply:
I agree, this probably shouldn't remove the newlines by default. I think this was just for improving formatting with output like CSV, because it wouldn't always turn out properly.

Which would be preferred, removing this `.replace()` call and making a possibly breaking change, or leaving as is?